File: aoj/volume100/php/A10010.php
<?php
while (true) {
    fscanf(STDIN, "%d %s %d", $a, $op, $b);
    if ($op === "?")
        break;

    $result = 0;
    switch ($op) {
    case "+":
        $result = $a + $b;
        break;
    case "-":
        $result = $a - $b;
        break;
    case "*":
        $result = $a * $b;
        break;
    case "/":
        $result = $a / $b;
        break;
    }
    printf("%d\n", $result);
}

File: aoj/volume100/python3/A10010.py
import sys

operators = {
    "+": lambda a, b: a + b,
    "-": lambda a, b: a - b,
    "*": lambda a, b: a * b,
    "/": lambda a, b: a // b,
    }

for line in sys.stdin:
    raw_a, op, raw_b = line.rstrip().split()
    if op == "?":
        break

    a, b = map(int, [raw_a, raw_b])
    result = operators[op](a, b)

    print(result)
